README.md

```diff
@@ -44,20 +44,20 @@ pip install --editable .
 ## Script Usage
 
 ```sh
-exhibitor-tls-artifacts [OPTIONS] [SANS]...
+Usage: exhibitor-tls-artifacts [OPTIONS] [NODES]...
+
+  Generates Admin Router and Exhibitor TLS artifacts. NODES should consist
+  of a space seperated list of master ip addresses. See
+  https://docs.mesosphere.com/1.13/security/ent/tls-ssl/exhibitor-tls/
 
-Args:
-    SANS: Subject Alternative Names to be put in the end-entity
-              certificates. Can be DNS names or IP addresses.
 Options:
-  -d, --dir TEXT  Directory to put generated artifacts in.
-                  Default: ./artifacts/ .
+  -d, --dir TEXT  Directory to put artifacts in.
```

Comment:
Let's extend the help text and state what happens if the directory exists (is it wiped? are individual files overwritten upon conflict?).

Reply:
👍